src/main/java/org/folio/circulation/domain/notice/schedule/ScheduledNoticesRepository.java

```diff
@@ -103,6 +111,15 @@ private ScheduledNoticesRepository(
       .after(this::deleteMany);
   }
 
+  public CompletableFuture<Result<Response>> deleteOverdueNotices(String loanId) {
+
+    return exactMatch(LOAN_ID, loanId)
+      .combine(exactMatch(TRIGGERING_EVENT, DUE_DATE.getRepresentation()), CqlQuery::and)
+      .combine(exactMatchAny(NOTICE_CONFIG + "." + TIMING,
+        Arrays.asList(UPON_AT.getRepresentation(), AFTER.getRepresentation())), CqlQuery::and)
```

Comment:
What `getRepresentation()` means here? Is it a json representation or constant name? 

Can we declare a constant list e.g. 
```java
UPON_AT_AND_AFTER_TAIMING = Arrays.asList(UPON_AT.getRepresentation(), AFTER.getRepresentation());
```

Reply:
> What getRepresentation() means here? Is it a json representation or constant name?

It's value's representation in the database. For example, UPON_AT is stored as "Upon At" in JSONB.

> Can we declare a constant list 

Done.